src/engine/Server.h

```diff
@@ -74,6 +81,7 @@ class Server {
   const size_t numThreads_;
   unsigned short port_;
   std::string accessToken_;
+  std::string gitHash_;
```

Comment:
The hash is a global constant embedded into the binary. There's no need to allocate a buffer here, so just using `std::string_view` suffices.